benchopt/plotting/helpers_compat.py

```diff
@@ -5,17 +5,17 @@
     go = None
 
```

Comment:
A gross translation like this migh be okay. We don't need exact equivalence, more a stable one no?

```suggestion

TRANSLATE_MARKERS = {v: i for i, v in enumerate(plt.Line2D.markers)}
```